turbinia/config/recipe_examples/test.yaml

```diff
@@ -0,0 +1,21 @@
+jobs_allowlist: 
+  - PlasoJob
+  - PsortJob
+
+patterns_file: '/home/jorlamd/.config/turbinia/patterns_files/patterns_test'
+
+plaso_base:
+  task: 'PlasoTask'
+  status_view: 'none'
```

Comment:
So, will this turn into a None object after being deserialized, or a string of 'none'?